Algebra.java

```diff
@@ -12,7 +12,7 @@ public static void main(String args[]) {
  		System.out.println(times(3,4));  // 3 * 4
```

Comment:
There was HW violation in this assignment.
I will clarify again: during the assignment, you are allowed to use only things you were taught in Lectures or Recitations, unless you were specifically told otherwise.

Each of those violation cause deduction of points..
Total deductions: 2 points.

Talk to me,
Tal Danai